libs/permit-utils/src/abi/erc20.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "constant": true,
+    "inputs": [],
+    "name": "name",
+    "outputs": [
+      {
+        "name": "",
+        "type": "string"
+      }
+    ],
+    "payable": false,
+    "stateMutability": "view",
+    "type": "function"
+  }
+]
```

Comment:
It's a duplication, I know, but I want to have just the minimal ABI possible.
Also, if I would use the one from the abis lib, it would create an inner dependency and I don't want that for this lib